pages/common/flake8.md

```diff
@@ -0,0 +1,20 @@
+# flake8
+
+> Tool to check the style and quality of Python code.
+> More information: <https://flake8.pycqa.org/>.
+
+- Lint a file or directory:
```

Comment:
```suggestion
- Lint a file or directory recursively:
```
Maybe it's obvious, so only apply if you want to.

Reply:
Good idea! Will apply.